.readme-partials.yml

```diff
@@ -8,6 +8,9 @@ introduction: |-
         [Pub/Sub quickstart](https://cloud.google.com/pubsub/docs/quickstart-client-libraries),
         [publisher](https://cloud.google.com/pubsub/docs/publisher), and [subscriber](https://cloud.google.com/pubsub/docs/subscriber)
         guides.
+
+        A fairly comprehensive list of changes in each version may be found in
```

Comment:
It feels like this would have generalized value linked from the README.  What do you think of adding a section to the README in synthtool so everyone in the class gets this?

Reply:
I wasn't sure if it should get templated into synthtool or not. I can do a PR for that if it seems like something everyone would want to have.